### ISSUE TITLE
./bam2msa: cannot excute binary file

### ISSUE THREAD

**Issue by uliyt** [orangeSi/bam2msa#3]
Hi @orangeSi , I have cloned the repo, and tried the command below.

`$ ./bam2msa
-bash: ./bam2msa: cannot execute binary file`

As you can see in the message, `bam2msa` is not executable. Is there any instructions for installation,  as what I read in `README.md` file is a tutorial to use it, no information on installation, or I missed important part in any of the file?

**Comment by Aciole-David** [comment]
I guess the problem is your bash syntax
Can you please detail more about it?

**Comment by uliyt** [comment]
Hi @Aciole-David , thank you for your prompt comment, the error message confused me too.

After I cloned the repo, I run the `./bam2msa` and got the message above.

Nothing special in my sourced `.bashrc` file. Currently, the reasons for this issue I can think of are:

1. weird rules in my `.bashrc`. Ruled out. I have tried to run `./bam2msa` before I source `.bashrc` and got the same message.
2. `bam2msa` was compiled for specific Linux system such as Centos, so it won't work on Ubuntu or RedHat or others. (I am not a computer guy, so not sure how far i went.)
